server/repository/src/diesel_macros.rs

```diff
@@ -140,6 +144,10 @@ macro_rules! apply_simple_string_filter {
                 $query = $query.filter($dsl_field.eq(value));
             }
 
+            if let Some(value) = string_filter.insensitive_equal_to {
+                $query = $query.filter($dsl_field.ilike(value.replace("%", "")));
+            }
+
             if let Some(value) = string_filter.like {
                 // Use case insensitive like
                 $query = $query.filter($dsl_field.ilike(format!("%{}%", value)));
```

Comment:
I wonder if we should lift the "%" syntax out of the filter, i.e. the let the user in control to add "%" where needed?

Reply:
I was wondering similarly, particularly when searching for patients by name, it might be useful to expand your search, if you know how to do that. There is a similar functionality in mSupply 
Just to check what you are saying though - do you mean remove the `replace` from the `insensitive_equal_to` or take the % out of the `like`?

Reply:
I meant taking the % out of the like filter an make it behave like in a DB. Not sure how many changes this would involve though...

Reply:
yeah..  on both counts. I think it could be helpful to give control to the caller.. and could cause a lot of changes! Something to tuck away for later..?